lib/mayaUsd/fileio/utils/meshWriteUtils.h

```diff
@@ -29,11 +29,75 @@
 #include <pxr/usd/usdGeom/mesh.h>
 #include <pxr/usd/usdUtils/sparseValueWriter.h>
 
+#include <maya/MBoundingBox.h>
 #include <maya/MDagPath.h>
 #include <maya/MFnMesh.h>
 #include <maya/MObject.h>
 #include <maya/MString.h>
 
+/**
+ * Gets the minimum unique name of a DAG node.
+ *
+ * @param node  The node to find the name of.
+ *
+ * @return      The name as a Maya string.
+ */
+MString mayaGetUniqueNameOfDAGNode(const MObject& node);
+
+/**
+ * Finds a child plug with the given `name`.
+ *
+ * @param parent    The parent plug to start the search from.
+ * @param name      The name of the child plug to find. This should be the short name.
+ *
+ * @return          The plug if it can be found, or a null `MPlug` otherwise.
+ */
+MPlug mayaFindChildPlugWithName(const MPlug& parent, const MString& name);
```

Comment:
This seems generally useful and not specific to meshes at all, so I'd suggest that this go in `mayaUsd/utils/util.[h,cpp]`, maybe next to the existing `GetPlugByName()` as `GetChildPlugByName()`:
https://github.com/Autodesk/maya-usd/blob/1439297b4ba79bdcffcfa6551b143ea7ac46f99a/lib/mayaUsd/utils/util.h#L175

Reply:
Addressed in 6460336.